src/content/docs/Language Common/strings.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Strings
+description: Strings
+sidebar:
+    order: 62
+---
+In C3, multiple string types are available, each suited to different use cases.
+
+#### String
+
+```c3
+
+distinct String = inline char[];
+
+```
+\
+Strings are usually the typical type to use, they can be sliced , compared etc ... \
```

Comment:
Strings are the typical string type to use, they can be sliced, compared, split etc, [the string methods are outlined here](#string-member-functions).
It is possible to access the length of a `String` instance through the  `.len`  member, eg
```c3
String my_string = "example";
int length = my_string.len
```